src/vscode-cosmos.api.d.ts

```diff
@@ -54,7 +59,7 @@ export interface DatabaseTreeItem extends DatabaseAccountTreeItem {
 
 export type CosmosDBResourceType = 'DatabaseAccount' | 'Database';
 
-export type CosmosDBApiType = 'Mongo' | 'SQL' | 'Graph' | 'Table';
+export type CosmosDBApiType = 'Mongo' | 'SQL' | 'Graph' | 'Table' | 'Postgres';
```

Comment:
I guess one option is detect the version and throw an error for any older versions, have the user upgrade to new version. Another option could be to include support for older versions in the code; which possible requires checks at several places of the code. 

But, do we need to support older versions tho given we are trying to rebrand?

Reply:
Ideally users who have previously connected a mongo database would still see that database in the tree (even if the UI is slightly different). I think the "rebrand" is more for new users who previously didn't know that we supported postgres

Reply:
This is the current status of version difference: 
Older App Service + Latest Databases
- Can see all postgres accounts when trying to add connections on App Service, both attached and azure
- However, no appsettings support and no creation of a treeitem in case user does add a postgres connection, it will be added as a MongoURL appsetting and treeitem would not be created

Latest App Service + Older Databases
- Cannot see any postgres accounts when trying to add connections on App Service

The first scenario could be adjusted if needed, but I don't see any harm it keeping it that for this release.